src/shadow-tlb.h

```diff
@@ -109,6 +109,19 @@ static inline uint64_t tlb_get_entry_cold_abs_addr(uint64_t eidx) {
         (ETYPE * sizeof(std::array<tlb_cold_entry, PMA_TLB_SIZE>)) + (eidx * sizeof(tlb_cold_entry));
 }
 
+template <TLB_entry_type ETYPE>
+static inline uint64_t tlb_get_entry_hot_rel_addr(uint64_t eidx) {
```

Comment:
Can you simplify `tlb_get_entry_hot_abs_addr` to make use of this function, so this code is covered and not dead-code?

Then do the same for `tlb_get_entry_cold_abs_addr`.

Reply:
done! good idea